pkg/kubelet/network/dns/dns_windows.go

```diff
@@ -63,11 +64,18 @@ var (
 	procGetNetworkParams = iphlpapidll.MustFindProc("GetNetworkParams")
 )
 
+func fileExists(filename string) bool {
+	stat, err := os.Stat(filename)
+	if os.IsNotExist(err) {
+		return false
+	}
+	return stat.Mode().IsRegular()
```

Comment:
If you get any error other than `IsNotExist()`, you can't rely on `stat`.  You need to handle other errors.

Reply:
Done